eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -146,6 +146,12 @@ jobs:
       VcpkgDependencies: curl[winssl] libxml2 nlohmann-json
       VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
     steps:
+      - script: npm i -g cspell
+        displayName: Install cspell
+
+      - script: cspell lint --config cspell.json --no-summary "sdk/${{ parameters.ServiceDirectory }}/**/*"
```

Comment:
There are lots of spell checkers available.  Is each repro using its own language specific spell checker?  Is there a spell checker we could share across all repos?